### ISSUE TITLE
may be jsx-closing-tag-location is a bad rule

### ISSUE THREAD

**Issue by zhangyuang** [standard/standard#1447]
**What version of this package are you using?**
v14.0
**What operating system, Node.js, and npm version?**
10.0
**What happened?**

when you write ternary expression， jsx-closing-tag-location is very easy confilct with jsx-indent like this code
```
 {
                    foo == 0 ? <Col>
                      <FormItem>
                     test
                      </FormItem>
                    </Col> : null
 }
```
it cause standard can't fix the code style and save loop format my code
**What did you expect to happen?**
remove jsx-closing-tag-location, so i have been used standard as v13.0
**Are you willing to submit a pull request to fix this bug?**

**Comment by sheerun** [comment]
If https://github.com/eslint/eslint/pull/12556#issuecomment-554220017 is accepted to eslint and standard, this could be compatible with jsx-closing-tag-location with following code:

```es6
foo == 0
  ? <Col>
      <FormItem>
        test
      </FormItem>
    </Col>
  : null
```

**Comment by sheerun** [comment]
Alternatively:

```es6
foo == 0 ? (
  <Col>
    <FormItem>test</FormItem>
  </Col>
) : null
```

**Comment by feross** [comment]
This issue was fixed in `standard` 15.0.1. The rules no longer conflict anymore.
